### PR TITLE
Revert "speed tests up by parallelising them"

### PR DESCRIPTION
**Pull request by fabiosantoscode**
This reverts commit 58d6d326c32d754d46db23a78b0dd25166030fd7.

### CODE REVIEW

**Comment by fabiosantoscode** [comment]
This should fix the mocha issue on 0.10 @kzc, let's pray ;) I had to spend a lot of time bisecting to find this commit.

**Comment by fabiosantoscode** [comment]
This build: https://travis-ci.org/terser-js/terser/jobs/447429699 proves that it works. I removed the compress tests to see if the mocha tests would run.

**Comment by kzc** [comment]
> This reverts commit 58d6d32

I think it's for the best. It didn't significantly speed up Travis testing and had potential for error.

The open question is whether to test the bundle or the source directly:

```patch
-var U = require("../dist/bundle");
+var U = require("./node");
```

One can make the case for testing the same code that users will run in the release - namely, the bundle.

**Comment by fabiosantoscode** [comment]
Before each test, npm run prepare is run, so the bundle will always be fresh.

**Comment by kzc** [comment]
> Before each test, npm run prepare is run, so the bundle will always be fresh.

But the bundle will not be used in ` test/run-tests.js` with this PR for compress tests:

https://github.com/terser-js/terser/pull/143/files#diff-f30e37550fb6c2097774ae1a933aa36eR5

It would be ideal to run all tests against the same code that users will use in the release.

**Comment by fabiosantoscode** [comment]
@kzc adjusted the requires to use the bundle

**Comment by kzc** [comment]
@fabiosantoscode I'm not sure what you did in your last commit, but nearly every line with `../../dist/bundleTest` has a problem... https://github.com/terser-js/terser/pull/143/files

Please run "npm test" locally to verify before `git push`.

**Comment by fabiosantoscode** [comment]
Pushing a fix @kzc

**Comment by kzc** [comment]
@fabiosantoscode Something is still wrong with this PR. Not only is it rebased to latest master, but it also appears to have merged the unfinished work-in-progress PR #144.

Consider aborting and closing this PR and starting again after #144 is fixed and merged to master.

File: lib/compress.js [hunk 1]
```diff
@@ -4926,7 +4926,7 @@ merge(Compressor.prototype, {
             stat = null;
             for (var i = 0; i < len; i++) {
                 var line = body[i];
-                if (line instanceof AST_Var) {
+                if (line instanceof AST_Definitions) {
```
Comment (kzc) [inline lib/compress.js]: Why are there remnants of early (buggy) versions #120 in this PR?

**Comment by fabiosantoscode** [comment]
@kzc found those commits and rebased them off. Nice catch

**Comment by fabiosantoscode** [comment]
Build is green. Good to merge?

**Comment by kzc** [comment]
Go for it.

There were a couple of `npm install` failures [here](https://travis-ci.org/terser-js/terser/builds/447513995?utm_source=github_status&utm_medium=notification) but it's probably a Travis CI issue.

Are there any tests run in parallel within a given Travis CI process now?

**Comment by fabiosantoscode** [comment]
No, parallel testing is off.

**Pull request merged**